packages/accounts-base/url_client.js

```diff
@@ -86,21 +86,7 @@ AccountsTest = {
 // this is prevented by the need to set autoLoginEnabled=false, but in
 // some bright future we won't need to do that anymore.
 
-/**
- * @summary Register a function to call when a reset password link is clicked
- * in an email sent by
- * [`Accounts.sendResetPasswordEmail`](#accounts_sendresetpasswordemail).
- * This function should be called in top-level code, not inside
- * `Meteor.startup()`.
- * @param  {Function} callback The function to call. It is given two arguments:
- *
- * 1. `token`: A password reset token that can be passed to
- * [`Accounts.resetPassword`](#accounts_resetpassword).
- * 2. `done`: A function to call when the password reset UI flow is complete. The normal
- * login process is suspended until this function is called, so that the
- * password for user A can be reset even if user B was logged in.
- * @locus Client
- */
+// Documentation for this method can be found in accounts_client.js.
```

Comment:
I would prefer putting it back here with the `@memberof` annotation.

The main reason we switched to using JSDoc in the first place was to keep the documentation in the same place as the code, and it would be a real shame to break that property to avoid adding a tag to JSDoc, in my opinion.